src/api_recievers/influxqueries.js

```diff
@@ -308,4 +308,50 @@ const grabAllPreviousStudySessionIDs = () => {
 };
 
 
-module.exports = { appData, SpecificStudySessionProcessing, grabTimesForApp, insertStudySessionData, grabAllPreviousStudySessionIDs, connectToInflux };
\ No newline at end of file
+const deleteStudySession = async (sessionId) => {
+  const deleteAPI = new DeleteAPI(influxClient);
+
+  const start = '1970-01-01T00:00:00Z'; // Start of the range (earliest possible time)
+  const stop = new Date().toISOString(); // End of the range (now)
+  const deleteKey = `_measurement="AppChange" AND QuerySession == ${sessionId}`;
+  let prevSessionDeleted = null; // Initialize outside the loop
+  
+  const headers = {
+    'Authorization': process.env.INFLUXDB_TOKEN,
+    'Content-type': 'application/json'
+  };
```

Comment:
Based on [this example](https://github.com/influxdata/influxdb-client-js/blob/master/examples/delete.ts) I'm pretty sure we just don't need this header object, as the token already gets stored in influxClient

Reply:
Yes, I'm trying to figure out how to get token into the function call.